main.tf

```diff
@@ -0,0 +1,142 @@
+# Define composite variables for resources
+resource "null_resource" "default" {
```

Comment:
Can you refactor this into `tf_label` 

Since we use this pattern all over the place, it worries me to hardcode it in each module.